.travis.yml

```diff
@@ -17,7 +22,7 @@ before_script:
   - sh -e /etc/init.d/xvfb start
 
 script:
-  - BASEURL=http://127.0.0.1:9500 make all
+  - BASEURL=http://127.0.0.1:9500 PLATFORM=$PLATFORM BROWSER=$BROWSER make all
```

Comment:
...and the matrix only affects the settings for the wct and system-test portions